lib/screen/customer_support/support_thread_page.dart

```diff
@@ -748,6 +795,8 @@ class _SupportThreadPageState extends State<SupportThreadPage> {
           payload.artworkReportID != null &&
           payload.artworkReportID!.isNotEmpty) {
         data.artworkReportID = payload.artworkReportID;
+      } else if (payload is NewIssueFromAnnouncementPayload) {
```

Comment:
too many condition payload is {payload_type}, should change to use switch case when it more than 2 else if condition.